docs/program/apis/data-client.md

```diff
@@ -29,9 +29,8 @@ Data client API methods are only available in the Python SDK.
 
 To use the Viam data client API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`DataClient`](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient).
 See the following example for reference.
-To find the location secret, go to [Viam app](https://app.viam.com/), and go to the [**Code sample**](https://docs.viam.com/manage/fleet/robots/#code-sample) tab of any of the robots in the location.
-Toggle **Include secret** on and copy the `payload`.
-For the URL, use the address of any of the robots in the location (also found on the **Code sample** tab).
+To find the api key and secret, go to [Viam app](https://app.viam.com/), and navigate to the bottom of your location's page.
```

Comment:
```suggestion
To find the api key and secret, go to the [Viam app](https://app.viam.com/), and navigate to the bottom of your location's page.
```
But think this will likely need to be changed to `Security tab` if screenshot at top is accurate?